### PR TITLE
[Update] update credix dependency following onchain upgrade

### PR DESCRIPTION
**Pull request by uxd-vincent**
Updating the binaries of the smart contract following credix devnet on-chain update.
Switching to using the same program ID on mainnet/devnet.

### CODE REVIEW

File: programs/uxd/tests/integration_tests/api/program_test_context/process_instruction.rs [hunk 1]
```diff
@@ -35,8 +35,24 @@ async fn process_instruction_result(
...
-    result
+    // Done, try to print result and metadata
+    if raw_result.is_ok() {
+        let result_with_metadata = raw_result.unwrap();
+        let metadata_option = result_with_metadata.metadata;
+        if metadata_option.is_some() {
+            let metadata = metadata_option.unwrap();
+            println!(
+                " - instruction.compute_units_consumed: {:?}",
+                metadata.compute_units_consumed
```
Comment (uxd-vincent) [inline programs/uxd/tests/integration_tests/api/program_test_context/process_instruction.rs:46]: just extracting more data in the tests logs, for easier debug

File: .github/workflows/ci-cargo-audit.yml [hunk 2]
```diff
@@ -32,4 +32,4 @@ jobs:
       # Run cargo audit using args from .cargo/audit.toml (ignores, etc.)
       - name: Run Cargo Audit
         # Ignoring warnings from solana_program_test dependencies which are only in dev dependencies
-        run: cargo audit -c always --ignore RUSTSEC-2021-0139 --ignore RUSTSEC-2023-0001 --ignore RUSTSEC-2020-0071 --ignore RUSTSEC-2023-0034
+        run: cargo audit -c always --ignore RUSTSEC-2023-0001 --ignore RUSTSEC-2020-0071
```
Comment (uxd-vincent) [inline .github/workflows/ci-cargo-audit.yml:35]: Turns out updating the cargo.lock dependencies resolves a bunch of RUSTSEC errors

File: tests/cases/registerCredixLpDepositoryTest.ts [hunk 3]
```diff
@@ -32,9 +32,9 @@ export const registerCredixLpDepositoryTest = async function ({
         payer ?? authority,
         controller,
         depository,
-        uiAccountingSupplyRedeemableSoftCap,
         accountingBpsStampFeeMint,
-        accountingBpsStampFeeRedeem
+        accountingBpsStampFeeRedeem,
+        uiAccountingSupplyRedeemableSoftCap
```
Comment (uxd-vincent) [inline tests/cases/registerCredixLpDepositoryTest.ts:37]: parameters were accidentally inverted there

File: programs/uxd/Cargo.toml [hunk 4]
```diff
@@ -36,10 +36,7 @@ mercurial-vault = { git = "https://github.com/mercurial-finance/vault-sdk", feat
   "cpi",
 ], rev = "3724175d8435f7acb7a33077a5a9f1eb58ab3399" }
 
-# credix_client = { version="0.6.0" } # mainnet
-credix_client = { version = "0.6.0", features = [
-  "pre-mainnet",
-] } # devnet and CI
+credix_client = { version = "0.7.0" }
```
Comment (uxd-vincent) [inline programs/uxd/Cargo.toml:39]: this is nice, we now dont need to use a special program ID for only devnet debugging, we can use the same program as the credix team now

File: tests/utils.ts [hunk 5]
```diff
@@ -276,7 +276,7 @@ export async function createCredixLpDepositoryDevnetUSDC(): Promise<CredixLpDepo
       ),
       collateralSymbol: 'USDC(CredixDevnet)',
       credixProgramId: new PublicKey(
-        'CRdXwuY984Au227VnMJ2qvT7gPd83HwARYXcbHfseFKC'
+        'CRDx2YkdtYtGZXGHZ59wNv1EwKHQndnRc1gT4p8i2vPX'
```
Comment (uxd-vincent) [inline tests/utils.ts:279]: main change kinda

**Review by acamill — APPROVED** [review]

**Pull request merged**